api/src/test/java/marquez/db/migrations/V44_2__BackfillAirflowParentRunsTest.java

```diff
@@ -35,6 +31,7 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 
 @ExtendWith(MarquezJdbiExternalPostgresExtension.class)
+@FlywayTarget("44")
```

Comment:
_Minor_: Mind adding a comment as to why the test is fixed on migration `44`?

Reply:
Done